store/tikv/coprocessor.go

```diff
@@ -466,10 +466,17 @@ func (it *copIterator) sendToRespCh(resp copResponse, respCh chan copResponse) (
 	return
 }
 
+type copResultSubset struct {
```

Comment:
Add comments for this structure like "copResultSubset implements `kv.ResultSubset` interface."